tailwind.config.js

```diff
@@ -1,40 +1,36 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
+  darkMode: ["class"],
   content: [
-    "./src/pages/**/*.{js,ts,jsx,tsx}",
-    "./src/components/**/*.{js,ts,jsx,tsx}"
-  ],
+    './pages/**/*.{ts,tsx}',
+    './components/**/*.{ts,tsx}',
+    './app/**/*.{ts,tsx}',
+    './src/**/*.{ts,tsx}',
+	],
   theme: {
-    extend: {
-      spacing: {
-        "max-screen-size": "1920px"
-      },
-      gridTemplateColumns: {
-        autodesktop: "repeat(auto-fit, minmax(410px, 1fr))",
-        automobile: "repeat(auto-fit, minmax(300px, 1fr))"
-      },
+    container: {
+      center: true,
+      padding: "2rem",
       screens: {
-        xs: "425px",
-        // => @media (min-width: 425px) { ... }
-        "2xl": "1440px",
-        // => @media (min-width: 1440px) { ... }
-        "3xl": "1920px",
-        // => @media (min-width: 1920px) { ... }
-        "pass-max-screen": "1921px"
-        // => @media (min-width: 1921px) { ... }
+        "2xl": "1400px",
+      },
+    },
+    extend: {
+      keyframes: {
+        "accordion-down": {
+          from: { height: 0 },
+          to: { height: "var(--radix-accordion-content-height)" },
+        },
+        "accordion-up": {
+          from: { height: "var(--radix-accordion-content-height)" },
+          to: { height: 0 },
+        },
       },
-      fontFamily: {
-        inter: ["Inter", "sans-serif"]
```

Comment:
Not sure if this matters but before the implementation of ShadCN UI, the text used for the website was the default font established by the browser and not the Inter font. So visually there's no change font-wise across the site, but maybe it might be good to use the Inter font with NextJS so that you can use next/font across the website: https://nextjs.org/docs/pages/api-reference/components/font